xbmc/cores/AudioEngine/Sinks/AESinkStarfish.cpp

```diff
@@ -19,6 +19,14 @@ using namespace std::chrono_literals;
 
 namespace
 {
+// clang-format off
+constexpr std::array<unsigned int, 3> defaultSampleRates = {
```

Comment:
Missing `#include <array>`

Reply:
Seems to be included from #include "utils/Map.h". I can add #include <array> here if that is what is required?